### DISCUSSION TITLE
Data about two wind turbines

### DISCUSSION THREAD

**Discussion by danielemeschino** [OpenFAST/openfast#2285] [Q&A]
Hello, I'm looking for the twist distribution or input data for openfast of the turbines UP77-1500 and AW77-1500 for my master thesis, someone can help me?

**Comment by jjonkman** [comment]
Dear @danielemeschino,

I'm not personally familiar with these turbines and doubt that there is an OpenFAST model of them available in the public domain.

Best regards,
